README.md

```diff
@@ -45,6 +45,21 @@ When the ```docker-compose.watch.yml``` file is included in the ```docker-compos
 This message will appear once every minute as the script continues to watch for new timesteps that may have been added. If there is an error message or you do not see the ```Fetching /shots/index.json``` message after the console output has slowed down or stopped, you can get help by creating an [issue](https://github.com/Kitware/eSimMon/issues/new) with the console output.
 
 
+Running the watch service separately
```

Comment:
Let call it the monitoring service ( above as well ). We would probably rename some files eventually.

Reply:
Fixed